src/pages/Payment.vue

```diff
@@ -90,7 +86,7 @@ export default {
     },
     methods: {
         showConfirmation: function() {
-            this.setMessage("Thanks, your payment request has been sent.", constants.MESSAGE_SUCCESS);
+            this.setMessage("Thanks, your payment request has been sent. Please take your vouchers to your market representative for them to be sent off.", constants.MESSAGE_SUCCESS);
```

Comment:
I think we still need it to be a variable/ component though - because different markets have different instructions. See: https://trello.com/c/VZF4RYMN/185-add-market-specific-variations-for-pay-claim-voucher-return-instructions